mono/metadata/icall-def.h

```diff
@@ -400,28 +400,32 @@ ICALL_TYPE(IOSELECTOR, "System.IOSelector", IOSELECTOR_1)
 ICALL(IOSELECTOR_1, "Add", ves_icall_System_IOSelector_Add)
 ICALL(IOSELECTOR_2, "Remove", ves_icall_System_IOSelector_Remove)
 
-ICALL_TYPE(MATH, "System.Math", MATH_19)
-NOHANDLES(ICALL(MATH_19, "Abs(double)", ves_icall_System_Math_Abs_double))
-NOHANDLES(ICALL(MATH_20, "Abs(single)", ves_icall_System_Math_Abs_single))
-NOHANDLES(ICALL(MATH_1, "Acos", ves_icall_System_Math_Acos))
-NOHANDLES(ICALL(MATH_2, "Asin", ves_icall_System_Math_Asin))
-NOHANDLES(ICALL(MATH_3, "Atan", ves_icall_System_Math_Atan))
-NOHANDLES(ICALL(MATH_4, "Atan2", ves_icall_System_Math_Atan2))
-NOHANDLES(ICALL(MATH_21, "Ceiling", ves_icall_System_Math_Ceiling))
-NOHANDLES(ICALL(MATH_5, "Cos", ves_icall_System_Math_Cos))
-NOHANDLES(ICALL(MATH_6, "Cosh", ves_icall_System_Math_Cosh))
-NOHANDLES(ICALL(MATH_7, "Exp", ves_icall_System_Math_Exp))
-NOHANDLES(ICALL(MATH_8, "Floor", ves_icall_System_Math_Floor))
-NOHANDLES(ICALL(MATH_9, "Log", ves_icall_System_Math_Log))
-NOHANDLES(ICALL(MATH_10, "Log10", ves_icall_System_Math_Log10))
-NOHANDLES(ICALL(MATH_11, "Pow", ves_icall_System_Math_Pow))
-NOHANDLES(ICALL(MATH_12, "Round", ves_icall_System_Math_Round))
-NOHANDLES(ICALL(MATH_14, "Sin", ves_icall_System_Math_Sin))
-NOHANDLES(ICALL(MATH_15, "Sinh", ves_icall_System_Math_Sinh))
-NOHANDLES(ICALL(MATH_22, "SplitFractionDouble", ves_icall_System_Math_SplitFractionDouble))
-NOHANDLES(ICALL(MATH_16, "Sqrt", ves_icall_System_Math_Sqrt))
-NOHANDLES(ICALL(MATH_17, "Tan", ves_icall_System_Math_Tan))
-NOHANDLES(ICALL(MATH_18, "Tanh", ves_icall_System_Math_Tanh))
+ICALL_TYPE(MATH, "System.Math", MATH_1)
```

Comment:
This is unwanted line damage

Reply:
could you please clarify why?

Reply:
I believe @kumpera wants a smaller edit here. If you are inserting functions, you can extend the numbering scheme, e.g. insert "17a" between "17" and "18".

Reply:
Jay is on spot. Larger changes that make it hard to review and reduce the value of history for no  good reason are to be avoided.

Reply:
got it, now the diff is smaller, but looks a bit weird :)